src/HIE/Bios/Cradle.hs

```diff
@@ -229,6 +229,7 @@ getCabalWrapperTool = do
         return wrapper_fp
       else do
         writeSystemTempFile "bios-wrapper" cabalWrapper
+  -- TODO: This isn't portable for windows
```

Comment:
I think it is portable, because the definition comes from `unix-compat`, at least it works correctly?

Reply:
yeah, it does so it could be deleted (i kept the existing comment without too much thinking)